plugins/destination/test/go.mod

```diff
@@ -5,13 +5,14 @@ go 1.22.7
 require (
 	github.com/apache/arrow/go/v17 v17.0.0
 	github.com/cloudquery/codegen v0.3.21
-	github.com/cloudquery/plugin-sdk/v4 v4.71.1
+	github.com/cloudquery/plugin-sdk/v4 v4.72.0
```

Comment:
I'll hold off updating to this SDK version otherwise you'd have two arrow versions in the plugin, v17 and v18.

The update is done via https://github.com/cloudquery/cloudquery/pull/19963

Reply:
Downgraded. The reason I initially upgraded was to see if the streambatchwriter issue was already fixed.